.github/workflows/ci.yml

```diff
@@ -154,7 +154,7 @@ jobs:
       - name: Setup Java and Scala
         uses: olafurpg/setup-scala@v13
```

Comment:
I thought we weren't using setup-scala anymore

Reply:
I'm not sure how you stumbled upon this. It was not even in the diff, was that on a specific commit?

Reply:
no I think it was on the one big diff before you updated the base branch